adapters/Unity/UnityAdapter/GADMUnityInterstitialAd.m

```diff
@@ -118,73 +118,28 @@ - (void)presentInterstitialFromRootViewController:(UIViewController *)rootViewCo
     return;
   }
 
-  if (![UnityAds isReady:_placementID]) {
-    NSError *error = GADMAdapterUnityErrorWithCodeAndDescription(
-        GADMAdapterUnityErrorShowAdNotReady, @"Failed to show Unity Ads interstitial.");
-    [strongConnector adapter:strongAdapter didFailAd:error];
-    return;
+  if (!_placementID) {
+    NSLog(@"Unity Ads received call to show before successfully loading an ad");
   }
-
-  [UnityAds addDelegate:self];
+  
   [strongConnector adapterWillPresentInterstitial:strongAdapter];
-  [UnityAds show:rootViewController placementId:_placementID];
+  [UnityAds show:rootViewController placementId:_placementID showDelegate:self];
 }
 
-#pragma mark - Unity Delegate Methods
-
-- (void)unityAdsPlacementStateChanged:(NSString *)placementId
-                             oldState:(UnityAdsPlacementState)oldState
-                             newState:(UnityAdsPlacementState)newState {
-}
+#pragma mark - UnityAdsLoadDelegate Methods
 
-- (void)unityAdsDidFinish:(NSString *)placementID withFinishState:(UnityAdsFinishState)state {
-  [UnityAds removeDelegate:self];
+- (void)unityAdsAdLoaded:(nonnull NSString *)placementId {
+  _placementID = placementId;
```

Comment:
rename _placementID to _loadedPlacementID, as that is now how it's being used in this class.